### PR TITLE
Add project:serve-web

### PR DESCRIPTION
**Pull request by bhageena**
If the web folder has `package.json`, it  proxies to the current/given namespace APIs and runs `npm start` otherwise serves static content using 'npx http-server'
links #45
Usage:
`nim web serve`  defaults to the current project's web folder 
`nim web serve /projects/nim/demo-projects/election`
`nim web serve /exp/ncovstats`

### CODE REVIEW

File: src/commands/web/serve.ts
Comment (rabbah) [inline src/commands/web/serve.ts:61]: Is there a programmatic interface to use instead? Is this better than an internal api? I don’t know which is better so simply asking.

**Review by rabbah — COMMENTED** [review]
Cool.

File: src/commands/web/serve.ts
Comment (bhageena) [reply src/commands/web/serve.ts:61]: I think it's possible to do it using internal APIs or we might even add `http-server` as a dependency and use its APIs, let's get the user feedback first. I am keeping that avenue open for the next iteration.

**Review by joshuaauerbachwatson — APPROVED** [review]
Looks good.   Approving.  I am holding off on merge until I test it (will probably test and merge all three outstanding PRs together).

**Pull request merged**
